10up/gutenbridge
# Adds pagination and classic mode integration with Block Catalog

### Description of the Change

This PR improves the bulk conversion of blocks. On sites with a mix of Classic & Gutenberg blocks, performance is significantly improved by limiting the WP_Query to Classic Editor Blocks. The PR also adds supports for pagination to the CLI.

### How to test the Change

Paginaton Options:
- Run the CLI `start` subcommand with `--per_page` and `--page` to run in batches. 
- You should see output that indicates the Pagination

Block Catalog Integration:
- Install Block Catalog plugin on a site with Classic Editor content and index with `wp block-catalog index --reset`
- Then run the CLI `start` subcommand with the `--catalog` option to limit the query to Classic Editor tagged posts
- You should see only posts with Classic Editor content found by the Block Catalog are being converted

### Changelog Entry

Added - Block Catalog integration, and pagination support

### Credits

Props @dsawardekar

### Checklist:
